content/desktop/synchronized-file-sharing.md

```diff
@@ -0,0 +1,93 @@
+
+After creating a file share instance, any container using a bind mount that references a host filesystem location which matches the stated synchronized file share host location, is provided by the Synchronized file sharing feature. Bind mounts that don't satisfy this condition are passed to the normal virtual filesystem bind-mounting mechanism, for example VirtioFS or gRPC-FUSE.
+
+> **Important**
+>
+> Synchronized file sharing isn't available on WSL or when using Windows containers. 
+{ .important }
+
+## Set up - TBD
+
+
+- DB customer
+- Signed in to DD 
+- DD version 4.25
+
+## Create a file share instance 
+
+To create a synchronized file share:
+1. Navigate to **Settings** within Docker Desktop and then within the **Resource** section, select the **File sharing** tab. 
+2. In the **Synchronized file shares** section, select the **plus** icon.
+3. Select a host folder to share. The synchronized file share should initialize and be usable.
+
+File shares take a few seconds to initialize as files are copied into the Docker Desktop VM. During this time, the status indicator displays **Preparing**.
+
+When the status indicator displays **Watching for filesystem changes**, your files are available to the VM through all the standard bind mount mechanisms, whether that's `-vm` in the command line or specified in your `compose.yml` file.
+
+## Explore your file share instance
+
+The **Synchronized file shares** section displays all your file share instances and provides useful information about each instance including:
+- The origin of the file share content
+- A Status update
+- How much space each file share is using
+- The number of filesystem entry counts
+- The number of symbolic links
+- Which container(s) is using the file share instance
+
+Selecting a file share instance expands the dropdown and exposes this information.
+
+## Use `.syncignore`
+
+You can use a `.syncignore` file at the root of each file share, to exclude local files from your file share instance. It supports the same syntax as `.dockerignore` files and excludes, and/or re-includes, paths from synchronization. `.syncignore` files are ignored at any location other than the root of the file share.
+ 
+Some example of things you might want to add to your `.syncignore` file are:
+- ?
+
+In general, the contents of your `.syncignore` file should be similar to what you have in your `.gitignore` file.
+
+## Frequently asked questions
+
+### What is the difference between Synchronized file sharing and the Mutagen extension?
+
+Synchronized file sharing is essentially a replacement for the Mutagen extension. However, it provides an improved user experience and the ability to use a [`.syncignore` file](#use-syncignore). You also no longer have to set the default ownership on files as this is now handled automatically.
+
+## Known issues
```

Comment:
Took these from https://www.notion.so/dockerinc/Mutagen-file-sharing-5bc28424acf24e8bb378d6be413be29a

Felt these were the most pertinent for users, lemme know if I'm wrong

Reply:
We have a few known issues:

- (maybe not worth mentioning) Once you enable the EAP, you'll have to re-open the dashboard and navigate back to the File Sharing pane; it won't automatically re-open.
- `.syncignore` changes won't cause deletion until file share recreation; i.e. files that become ignored due to a modification to `.syncignore` are left in place, but no longer update through synchronization.
- Case conflicts (due to Linux being case-sensitive and macOS/Windows only being case-preserving) display as "file exists" problems in the GUI.
- We report ephemeral issues aggressively, so spurious `Conflict` and `Problem` indicators may appear in the GUI during synchronization.  These will eventually be hidden by future heuristics, but we don't want to mask anything during EAP.
- Mutagen file shares mounted into ECI containers are currently read-only
- The Delete button doesn't currently work during the initial synchronization; you have to wait for it to complete before pressing Delete will have any effect

Reply:
With: 

>- Case conflicts (due to Linux being case-sensitive and macOS/Windows only being case-preserving) display as "file exists" problems in the GUI.
>- We report ephemeral issues aggressively, so spurious Conflict and Problem indicators may appear in the GUI during synchronization. 

Whats the solution? Just to ignore?

Reply:
Oh sorry, good point.  The solution would be to ignore, unless they are persistent and/or long-lived, in which case we should ask that users file a ticket.

@smnovick Have we figured out the communication channels for filing EAP tickets yet?